icepyx/core/query.py

```diff
@@ -22,6 +21,7 @@
 import icepyx.core.spatial as spat
 import icepyx.core.temporal as tp
 from icepyx.core.visualization import Visualize
+from icepyx.core.auth import EarthdataAuthMixin
```

Comment:
```suggestion
from icepyx.core.auth import EarthdataAuthMixin
import icepyx.core.spatial as spat
import icepyx.core.temporal as tp
from icepyx.core.visualization import Visualize
```
use alphabetical order convention